bff/test/Bff.Tests/TestFramework/GenericHost.cs

```diff
@@ -12,179 +12,178 @@
+    }
 
-        public string Url(string? path = null)
-        {
-            path ??= String.Empty;
-            if (!path.StartsWith("/")) path = "/" + path;
-            return _baseAddress + path;
-        }
+    public string Url(string? path = null)
+    {
+        path ??= String.Empty;
+        if (!path.StartsWith("/")) path = "/" + path;
+        return _baseAddress + path;
+    }
 
-        public async Task InitializeAsync()
-        {
-            var hostBuilder = new HostBuilder()
-                .ConfigureWebHost(builder =>
-                {
-                    builder.UseTestServer();
+    public async Task InitializeAsync()
+    {
+        var hostBuilder = new HostBuilder()
+            .ConfigureWebHost(builder =>
+            {
+                builder.UseTestServer();
 
-                    builder.ConfigureServices(ConfigureServices);
-                    builder.Configure(app =>
+                builder.ConfigureServices(ConfigureServices);
+                builder.Configure(app =>
+                {
+                    if (UseForwardedHeaders)
                     {
-                        if (UseForwardedHeaders)
+                        app.UseForwardedHeaders(new ForwardedHeadersOptions
                         {
-                            app.UseForwardedHeaders(new ForwardedHeadersOptions
-                            {
-                                ForwardedHeaders = ForwardedHeaders.XForwardedFor |
-                                                   ForwardedHeaders.XForwardedProto |
-                                                   ForwardedHeaders.XForwardedHost
-                            });
-                        }
-
-                        ConfigureApp(app);
-                    });
+                            ForwardedHeaders = ForwardedHeaders.XForwardedFor |
+                                               ForwardedHeaders.XForwardedProto |
+                                               ForwardedHeaders.XForwardedHost
+                        });
+                    }
+
+                    ConfigureApp(app);
                 });
+            });
 
-            // Build and start the IHost
-            var host = await hostBuilder.StartAsync();
+        // Build and start the IHost
+        var host = await hostBuilder.StartAsync();
 
-            Server = host.GetTestServer();
-            BrowserClient = new TestBrowserClient(Server.CreateHandler());
-            BrowserClient.BaseAddress = new Uri(this._baseAddress);
-            HttpClient = Server.CreateClient();
-            BrowserClient.BaseAddress = new Uri(this._baseAddress);
-        }
+        Server = host.GetTestServer();
+        BrowserClient = new TestBrowserClient(Server.CreateHandler());
+        BrowserClient.BaseAddress = new Uri(this._baseAddress);
+        HttpClient = Server.CreateClient();
+        HttpClient.BaseAddress = new Uri(this._baseAddress);
```

Comment:
Why is this part of this PR?

Reply:
Looks like a botched rebase conflict.

Reply:
Re-did commits to exclude. @Erwinvandervalk that change in a separate PR pls.